OxfordBodleian/Juel-Jensen/BDLaethg35.xml

```diff
@@ -264,37 +265,53 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                                     <desc>Spaces for rubrication have been left unfilled on three opening,
                                         ( <locus from="6v" to="8r"/>, <locus from="15v" to="16r"/>,
                                         and <locus from="21v" to="22r"/>). Very rarely, several words have been erased (<locus target="#13v #20r"/>).</desc>
                                 </item>
                             </list>
                         </additions>
-                        
-                   <!--     <bindingDesc>
-                            <binding xml:id="binding">
-                                <decoNote xml:id="b1"></decoNote>
-                                <decoNote xml:id="b2" type="bindingMaterial">
-                           <material key="wood"/>
-                        </decoNote>
+
+                        <bindingDesc>
+                            <binding contemporary="Ethiopian">
+                                <decoNote xml:id="b1" type="bindingMaterial">
+                                  The book is bound in <material key="wood">wooden</material> boards.
+                                </decoNote>
+                                <decoNote xml:id="b2" type="Boards">
+                                  The boards are flush with the bookblock an have shallow external bevels. The grain runs vertically.
+                                  The external surface of the boards is polished, while the inner surface exhibits signs of the working tool.
+                                </decoNote>
+                                <decoNote xml:id="b3" type="SewingStations">3>
+                                  Chainstitch sewing on three sewing stations; continuous fold pattern; Z-plied thread.
+                                  The bookblock was probably originally sewn on two sewing stations as the boards present only two sites for board attachment.
+                                  The manuscript has been resewn adding a third sewing station at a short distance from the second one. The quires also present an unused sewing hole above the first sewing station.
+                                  The ends of the thread are knotted in the centrefold of the last quire.
+                                </decoNote>
+                                <decoNote xml:id="b4" type="SlipCase">
+                                  The manuscript is provided with a leather one-piece single-walled slip case with the remnant of a hinging strap.
+                                  The case is equipped with a long flap extending from the left side panel and a main pointed flap extending from the back to the front
+                                  panel to cover the opening of the book compartment.
+                                  The fragment of a narrow leather hanging strap passes through two flaps extending from the bottom panel of the case.
+                                  outer case.
```

Comment:
There is something wrong in the end of this sentence

Reply:
You are right!!